istioctl/pkg/install/verify.go

```diff
@@ -122,10 +151,46 @@ func verifyPostInstall(enableVerbose bool, istioNamespaceFlag string,
 			for _, c := range job.Status.Conditions {
 				if c.Type == v1batch.JobFailed {
 					msg := fmt.Sprintf("Istio installation failed, incomplete or"+
-						" does not match \"%s\" - the required Job  %s failed", options.Filenames[0], name)
+						" does not match \"%s\" - the required Job %s failed", filename, name)
 					return errors.New(msg)
 				}
 			}
+		case "IstioOperator":
+			result := info.Client.
```

Comment:
Does this mean we will always use the `IstioOperator` stored in cluster other than the one specified via -f ? 
I think we should do like: 
if no `-f` specified, we will perform post install verification using the IstioOperator stored in cluster.
if `-f` specified, we will use it directly to generate manifest and do verify.